docs-website/sphinx/Makefile

```diff
@@ -22,7 +22,7 @@ $(VENV_SENTINEL): requirements.txt
 	$(VENV_DIR)/bin/pip install -r requirements.txt
 	touch $(VENV_SENTINEL)
 
-.PHONY: help html doctest linkcheck clean serve md
+.PHONY: help html doctest linkcheck clean clean_all serve md
```

Comment:
Can remove `clean` from here since `clean_all` covers that

Reply:
yeah - doesn't hurt to have it, but agree that it's redundant

Reply:
actually this is the `.PHONY` rule so it's required here